packages/coding-standards/src/CsFixer/Phpdoc/MissingParamAnnotationsFixer.php

```diff
@@ -67,7 +67,7 @@ protected function processFunctionToken(Tokens $tokens, int $index, ?Token $docT
             $newLines[] = new Line(sprintf(
                 '%s * @param %s %s%s',
                 $indent,
-                $type,
+                $type ?: 'mixed',
```

Comment:
This fixes empty type to `mixed`.

In @return it doesn't make sense (could be void), so there is no type.